.github/workflows/release.yml

```
@@ -0,0 +1,109 @@
+    - name: Extract metadata
+      id: meta
+      uses: docker/metadata-action@v5
+      with:
+        images: |
+          ghcr.io/${{ github.repository }}
+        tags: |
+          type=ref,event=branch
+          type=ref,event=pr
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}.{{minor}}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)